src/modules/registration/index.js

```diff
@@ -8,63 +8,94 @@ class Registration extends PureComponent {
   constructor(props) {
     super(props);
     this.state = {
+      name: '',
+      surname: '',
       username: '',
       password: '',
+      confirmedPassword: '',
       mail: ''
     }
   };
 
+  handleName = (name) => {
+    this.setState({ name });
+  };
+
+  handleSurname = (surname) => {
+    this.setState({ surname });
+  };
+
   handleUsername = (username) => {
-    this.setState({
-      username
-    });
+    this.setState({ username });
   };
 
   handlePassword = (password) => {
-    this.setState({
-      password
-    });
+    this.setState({ password });
   };
 
   handleMail = (mail) => {
-    this.setState({
-      mail
-    });
+    this.setState({ mail });
+  };
+
+  handleConfirmedPassword = (confirmedPassword) => {
+    this.setState({ confirmedPassword });
   };
 
-  handlePasswordConfimation = (password) => {
+  onAddingNewUser = () => {
+    const { name, surname, username, password,
+      confirmedPassword, mail } = this.state;
     {
-      this.state.password === password
-        ? console.log('Good')
-        : console.log('Wrong')
+      (password === confirmedPassword)
```

Comment:
It's better not to evaluate functions from ternary. Better approach is to use if statement